libcrux-ml-kem/src/ind_cca.rs

```diff
@@ -196,6 +196,8 @@ fn encapsulate<
     (ciphertext, shared_secret_array)
 }
 
+#[cfg_attr(hax,hax_lib::ensures(|result|
```

Comment:
Let's remove this for this PR and focus it on hash functions.

Reply:
Removed!